modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java

```diff
@@ -93,7 +93,7 @@
 		).put(
 			"showEmptyOption",
 			GetterUtil.getBoolean(
-				ddmFormField.getProperty("showEmptyOption"), true)
+				ddmFormField.getProperty("showEmptyOption"))
```

Comment:
@ygorrodriguesliferay, consider the scenario in which do you have a Select From List field allowing multiple options to be selected, but even though the user won't fill out the field, hence it should have an empty value which is a valid. Does it make sense to you?

Can you please clarify to me what is the root cause of the UI issue mentioned by you in LPS-136527?

Thanks.

Reply:
Hi @natocesarrego. I've tested the case the user won't fill out the field and it worked ok, I tried to make the same behavior when there are less fields to be displayed.
The issue happens because of this if present on https://github.com/liferay/liferay-portal/blob/d75a749f503bcaa193680fefda7da441807476a3/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js#L291

Please tell me if some change is needed.

Reply:
Hey @ygorrodriguesliferay, from what I could understand from the issue and the code shared by you, the idea is not display the option `Choose an option` just when multiple selections are allowed in the Select From List, right?

After all, do not check any of the options can be seen as an equivalent to the empty value, is it correct?

In light of this understanding, I believe that we should solve this issue only on the front-end, performing something like:
```
useEffect(() => {
		let result = options.filter(
			(option) =>
				option.value &&
				option.label.toLowerCase().includes(query.toLowerCase())
		);

		if (showEmptyOption && !multiple) { <== PROPOSED CHANGE
			const emptyOption = {
				label: Liferay.Language.get('choose-an-option'),
				value: null,
			};

			result = [emptyOption, ...result];
		}

		setFilteredOptions(result);
		// eslint-disable-next-line react-hooks/exhaustive-deps
	}, [options, query]);
```
The code above is the code shared by you on your last comment with a fix proposal. Do you think it'll fix the issue?

If so, can you please send a new pull with this suggestion?

Thanks in advance.